lib/DataTables.pm

```diff
@@ -362,9 +240,128 @@ sub json {
         $output{'aaData'} = ''; #we don't want to have 'null'. will break js
     }
 
-    return encode_json \%output;
+    return \%output;
 }
 
+sub json {
+    my $self = shift;
+
+    my $output_href = $self->table_data;
+
+    return encode_json $output_href;
+} # /json
+
+
```

Comment:
Can you remove these extra lines?

Reply:
I'm not sure what you mean. Do you mean the 2 linebreaks after a sub?